src/sidebar/util/session.js

```diff
@@ -28,14 +31,15 @@ export function shouldShowSidebarTutorial(sessionState) {
  *
  * @param {boolean} isSidebar - is the app currently displayed in a sidebar?
  * @param {Profile} profile - User profile returned from the API
- * @param {Object} settings - app configuration/settings
+ * @param {HostSettings} settings
  * @return {boolean} - Tutorial panel should be displayed automatically
  */
 export function shouldAutoDisplayTutorial(isSidebar, profile, settings) {
   const shouldShowBasedOnProfile =
     typeof profile.preferences === 'object' &&
     !!profile.preferences.show_sidebar_tutorial;
-  const service = serviceConfig(settings) || {};
+
+  const service = serviceConfig(settings) || { onHelpRequestProvided: false };
   return (
     isSidebar && !service.onHelpRequestProvided && shouldShowBasedOnProfile
```

Comment:
Without the change to the object literal here, TS complains that `service` might not have an `onHelpRequestProvided` property as it may have been initialized to `{}`, which is inferred to be an object type with no properties.